network-interface/src/network.rs

```diff
@@ -37,6 +37,8 @@ pub trait Topic {
     const VALIDATE: bool;
 }
 
+pub const MIN_SUPPORTED_MSG_SIZE: usize = 1024 * 1024;
```

Comment:
Do you mean `MAX`?

Reply:
I had a hard time naming this constant.

I want to say that this is the minimum maximum message size of the underlying network. I.e. the network has to at least support messages of this size. I don't want to tell the underlying network that it has to limit the sizes to exactly this, just that it needs to support messages of this size.

Reply:
That's great. I'd suggest putting a rustdoc comment on it, explaining it exactly like that:

```rust
/// The network implementation has to at least support messages of this size
pub const MIN_SUPPORTED_MSG_SIZE: usize = 1024 * 1024;
```

Reply:
Thanks for the concise desription. Added.